monai/transforms/utility/array.py

```diff
@@ -112,6 +112,7 @@ def __call__(self, img: NdarrayOrTensor) -> NdarrayOrTensor:
         return img
 
 
+@deprecated(since="0.9", msg_suffix="use monai.transforms.EnsureChannelFirst instead.")
```

Comment:
Hi @wyli ,

I think the `since` should be `0.8`, could you please help confirm it?

Thanks in advance.

Reply:
About the message can be:
msg_suffix="use `MetaTensor` data type and `monai.transforms.EnsureChannelFirst` instead."
@wyli What do you think?

Thanks.

Reply:
I think 0.9 here is fine because it's a warning for 0.9.1... Do you think there are use cases that addchannel can achieve but ensure channel first can't, should we mention it?

Reply:
Hi @wyli ,

`AddChannel` can work for PyTorch Tensor and numpy array directly while `EnsureChannelFirst` can't, I am not sure whether users still want to use it after we introduced 0.9.1 with `MetaTensor`...

Thanks.

Reply:
ok thanks,then I think this PR is ready, I'll merge it.